modal/_container_exec.py

```diff
@@ -43,7 +43,7 @@ async def container_exec(task_id: str, command: List[str], *, pty: bool, client:
     except GRPCError as err:
         connecting_status.stop()
         if err.status == Status.NOT_FOUND:
-            raise NotFoundError(f"Container ID {task_id} not found")
+            raise NotFoundError(err.message)
```

Comment:
I'm changing the server to make it return multiple kinds of not found messages for improved UX.